about.html

```diff
@@ -0,0 +1,148 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./css/about.css" />
+    <link
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css"
+      rel="stylesheet"
+    />
+    <title>Imaginary Dance</title>
+  </head>
+  <body>
+    <header>
+      <div class="top-header">
+        <ul class="top-header-list">
+          <li>
+            <a href=""><i class="fab fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a href=""><i class="fab fa-twitter"></i></a>
+          </li>
+          <li><a href="">English</a></li>
+          <li><a href="Index.html">My Page</a></li>
+          <li><a href="">Logout</a></li>
+        </ul>
+      </div>
+      <nav>
+        <div class="modal">
+          <img src="./Image/Enabled.svg" class="close" alt="closeTag" />
+          <div class="modal-body">
+            <ul class="modal-menu">
+              <li><a href="index.html">Home</a></li>
+              <li><a href="#partners">Partners</a></li>
+              <li><a href="about.html">About</a></li>
+            </ul>
+          </div>
+        </div>
+        <div class="logo">
+          <img src="./image/logo.png" alt="logo" />
+        </div>
+        <div class="nav-hamburger">
+          <img src="./image/button.png" class="hamburger" alt="" />
+        </div>
```

Comment:
- [OPTIONAL] Kindly fix the nav at the top of both pages😎 for a better user experience✌